### PR TITLE
Make utils.isAddress more strict (must start with a0 unless known special address)

### PR DESCRIPTION
**Pull request by aion-kelvin**
utils.isAddress returns true if the input is any 64-character hex string.  

This PR makes it more strict -- addresses in Aion network begin with a0, aside from a list of accounts that were created before that rule was added.  Those account addresses are in a hard-coded white list.    The new logic is that isAddress returns true if the input is 64-character hex and either begins with a0 or is in the white list.

Unit tests pass; ran most of the integ tests, which pass (one of them was not run due to related issues with setting up kernel for it, but the tests that did run uses the `isAddress` logic).

### CODE REVIEW

File: test/utils.isAddress.js [hunk 1]
```diff
@@ -16,7 +16,11 @@ var tests = [
     { value: '0xE247a45c287191d435A8a5D72A7C8dc030451E9F', is: false },
     { value: '0xe247a45c287191d435a8a5d72a7c8dc030451e9f', is: false },
     { value: '0xE247A45C287191D435A8A5D72A7C8DC030451E9F', is: false },
-    { value: '0XE247A45C287191D435A8A5D72A7C8DC030451E9F', is: false }
+    { value: '0XE247A45C287191D435A8A5D72A7C8DC030451E9F', is: false },
+    { value: '0xP06f640ced8bd31eb9e191887adde74888e9ca31fd8545dae3ae896773ccbc4f', is: false },
+    { value: '0x006f640ced8bd31eb9e191887adde74888e9ca31fd8545dae3ae896773ccbc4f', is: false },
+    { value: '0xc00dcc9fe51c73767fad07cd4da990a8aa7487f40ba5718f711b4fdc09ae5b6e', is: true},
+    { value: 'c00dcc9fe51c73767fad07cd4da990a8aa7487f40ba5718f711b4fdc09ae5b6e', is: true}
 ];
```
Comment (AionJayT) [inline test/utils.isAddress.js:24]: Could you add one more valid address with 'a0' prefix in the test case?

**Review by AionJayT — COMMENTED** [review]
LGTM, only one comment for the test case suggestion.

File: test/utils.isAddress.js (hunk 1 above)
Comment (aion-kelvin) [reply test/utils.isAddress.js:24]: The test actually pulls in a few other addrseses from other tests, which are all a0 and valid.  Here is the full list of addresses it tsets:

```
      ✓ shoud test if value function () {} is address: false
      ✓ shoud test if value function anonymous(
) {

} is address: false
      ✓ shoud test if value function is address: false
      ✓ shoud test if value [object Object] is address: false
      ✓ shoud test if value 0xc6d9d2cd449a754c494264e1809c50e34d64562b is address: false
      ✓ shoud test if value c6d9d2cd449a754c494264e1809c50e34d64562b is address: false
      ✓ shoud test if value 0xE247A45c287191d435A8a5D72A7C8dc030451E9F is address: false
      ✓ shoud test if value 0xE247a45c287191d435A8a5D72A7C8dc030451E9F is address: false
      ✓ shoud test if value 0xe247a45c287191d435a8a5d72a7c8dc030451e9f is address: false
      ✓ shoud test if value 0xE247A45C287191D435A8A5D72A7C8DC030451E9F is address: false
      ✓ shoud test if value 0XE247A45C287191D435A8A5D72A7C8DC030451E9F is address: false
      ✓ shoud test if value 0xP06f640ced8bd31eb9e191887adde74888e9ca31fd8545dae3ae896773ccbc4f is address: false
      ✓ shoud test if value 0x006f640ced8bd31eb9e191887adde74888e9ca31fd8545dae3ae896773ccbc4f is address: false
      ✓ shoud test if value 0xc00dcc9fe51c73767fad07cd4da990a8aa7487f40ba5718f711b4fdc09ae5b6e is address: true
      ✓ shoud test if value c00dcc9fe51c73767fad07cd4da990a8aa7487f40ba5718f711b4fdc09ae5b6e is address: true
      ✓ shoud test if value 0xa0202797a7aff86fec1a5d8b7cacea276de5bcfc2e8b14878c9ba48d7d5330a0 is address: true
      ✓ shoud test if value 0xa0202797a7aff86fec1a5d8b7cacea276de5bcfc2e8b14878c9ba48d7d5330a0 is address: true
      ✓ shoud test if value 0xa09d19c066f341220d914414c42e5e796e7e4daf83a9abf4581b1670403bbd15 is address: true
      ✓ shoud test if value 0xa06f640ced8bd31eb9e191887adde74888e9ca31fd8545dae3ae896773ccbc4f is address: true

```

Comment (AionJayT) [reply test/utils.isAddress.js:24]: cool.

**Review by AionJayT — APPROVED** [review]

**Pull request merged**